website/flask_helpers.py

```diff
@@ -70,3 +75,84 @@ def dumps(self, obj, **kwargs):
 
     def loads(self, s, **kwargs):
         return json.loads(s, **kwargs)
+
+
+whitelisted_links = {
+    'OPEN_BUG_REPORT':
+        "https://github.com/hedyorg/hedy/issues/new?assignees=&labels=&template=bug_report.md&title=%5BBUG%5D",
+    'SLIDES_COM': 'https://slides.com',
+    'TEACHERS_MANUAL': 'https://www.hedy.org/for-teachers/manual',
+    'TEACHERS_MANUAL_FEATURES': 'https://hedy.org/for-teachers/manual/features',
+    'TEACHERS_MANUAL_FOR_TEACHERS': 'https://hedy.org/for-teachers/manual/preparations#for-teachers',
+    'FOR_TEACHERS': 'https://hedycode.com/for-teachers',
+    'DISCORD_SERVER': 'https://discord.gg/8yY7dEme9r',
+    'MAIL_TO_HELLO_HEDY': 'mailto: hello@hedy.org',
+    'DISCORD_VIDEO': 'https://www.youtube.com/watch?v=Lyz_Lnd-_aI',
+    'CUSTOMIZE_ADVENTURE': 'https://github.com/hedyorg/hedy/assets/80678586/df38cbb2-468e-4317-ac67-92eaf4212adc',
+}
+
+whitelisted_keys = {v for k in whitelisted_links.keys() for v in {f'#HREF_{k}#', f'#MD_{k}#', f'#SRC_{k}#'}}
+
+whitelisted_html_tags = ['a', 'b', 'em', 'strong', 'img']
```

Comment:
We also use an `iframe` for Felienne's conference in strangeloop: 
https://github.com/hedyorg/hedy/blob/7fdaf40397ce3240a2028fc2a83d9bcbd6f5a783/content/pages/en.yaml#L69-L70

Reply:
I am not sure why this iframe is part of the .yaml file and not in the .html file. My intention is move it to the page itself.